provider/etcd/etcd_client.go

```diff
@@ -150,21 +186,26 @@ func (ec *EtcdClient) Get(fp model.Fingerprint) (*types.Alert, error) {
 	ec.mtx.Unlock()
 	if err != nil {
 		level.Error(ec.logger).Log("msg", "Error getting alert from etcd", "err", err)
+		etcdOperationsTotal.With(prometheus.Labels{"operation": "get", "result": "error"}).Inc()
```

Comment:
MAY replace all these duplicated labels with some predefined values, this is to prevent from typo at compile time.